AddOns/BasicFinancials/app/src/codeunit/BasicFinancialsMgmt.Codeunit.al

```diff
@@ -0,0 +1,125 @@
+codeunit 20601 "Basic Financials Mgmt BF"
+{
+    Access = Internal;
+
+    var
+        AzureADLicensing: codeunit "Azure AD Licensing";
+        NotSupportedLicenseErr: Label 'To deploy the Basic Financials extension the Basic Financials license must be assigned to at least one user.';
+        NotSupportedLocalesErr: Label 'The Basic Financials Extension can only available in Denmark and Iceland.';
+        NotSupportedUserErr: Label 'The user who deploys the Basic Financials extension must have the Super User permission set.';
+        NotSupportedCompanyErr: Label 'The Basic Financials extension can only be deployed when exactly one company exists in the environment.';
+        AllProfileFilterTxt: Label 'MANUFACTURING|PROJECTS|SERVICES|WAREHOUSE|SHIPPING AND RECEIVING - WMS|SHIPPING AND RECEIVING|WAREHOUSE WORKER - WMS|PRODUCTION PLANNER|PROJECT MANAGER|DISPATCHER|SALES AND RELATIONSHIP MANAGER', Locked = true;
+        BFSKUIdTxt: Label '{66CAD104-64F9-476E-9682-3C3518B9B6ED}', Locked = true, Comment = 'Dynamics 365 Business Central Basic Financials';
```

Comment:
hm, this doesn't look like correct ID. Looking at System module the BasicPlanGUIDTxt = '{2ec8b6ca-ab13-4753-a479-8c2ffe4c323b}';

Reply:
I reciewed a mail from Fredrik Hietala (Microsoft), where it is determined which licenses to test for. If this is incorrect please reconcile it internally in Microsoft and then notify change.

**Fredrik Hietala (Microsoft)**
The 2 licenses are the following: 
•	Dynamics 365 Business Central Basic Financials: 66cad104-64f9-476e-9682-3c3518b9b6ed
•	Dynamics 365 Business Central Basic Financials from C5 SPLA (Qualified Offer): 4dce07fd-7b07-4fb5-8fb7-d49653f7bf30